main/zoe/api/Zoe-Intro.md

```diff
@@ -0,0 +1,109 @@
+# Introduction to Zoe
+
+
+**Note:** Before reading this, you should be familiar with ERTP basics. (**tyg todo**: Add link)
+
+
+This is an introduction to Zoe, Agoric's smart contract framework. The Zoe Guide (**tyg todo**: Add link) has more complete and detailed information, including concepts not covered here. In particular, here we simplify Zoe examples such that there are always two parties in a basic trading/swap contract. We do not discuss more complicated contracts with many participants, such as auctions, or simpler contracts, such as our AutomaticRefund one party demo contract.
+
+
+## What is Zoe?
+
+
+*Zoe* is itself a long lived and trusted contract that serves as a framework which runs other contracts.
+
+
+## How do Zoe contracts work?
+
+
+To start using a contract, you need to *install* it on Zoe. This is putting the contract’s code into Zoe. Multiple contracts can be installed on Zoe, and each contract can have multiple simultaneous active offers, each of which is handled by a separate contract instance. For clarity, in this document a *contract* is the installed code that defines how the contract works. A *contract instance* is an active execution of the contract code.
+
+
+Once installed on Zoe, a contract waits for an *offer* to be made. That starts a lifecycle for the offer and new contract instance of:
+1. Sending out *invites* to request counter-offers.
+2. Partie *redeeming* those invites and making a counter-offer.
+3. Determining when the offer *matches* a counter-offer.
+4. Either succeeding or failing to make the transaction. 
+
+
+An offer’s completion only affects that offer and the assets involved in it; essentially its contract instance. The contract it was made on keeps running, waiting for and processing new offers. 
+
+
+*Offer safety* is automatically enforced for parties in a contract; if an offer succeeds and completes, all parties get what they wanted. If it fails to complete successfully for any reason, parties keep what they offered.
+
+
+All parties in a Zoe contract receive a *seat*, which lets them take action in that contract. They also receive a *promise* for a future payout. The contract itself does not have direct access to any parties’ assets. The payments for an offer are all run through Zoe and its *escrow* capabilities.  
+
+
+Zoe has two distinct APIs. One is client/user-facing, with `zoe.` prefixes on its commands. The other is only exposed to contracts built on Zoe (its *contract facet*), with `zcf.` prefixes on its commands. . 
+
+
+## Starting a contract on Zoe
+
+
+While there can be more than one Zoe instance installed and running at any time, for now we’ll focus on the single instance running on Agoric’s testnet. Of course, before you can run a smart contract on Zoe, you have to install it there. For Zoe purposes, *installation* means installing the code your contract is written in (JavaScript and of course the ERTP and Zoe API commands we’ve defined) on a Zoe instance. 
```

Comment:
I think we don't need to explain that there could be more than one Zoe instance running. That won't be the case for most basic uses and is confusing.